packages/amplication-server/src/core/dbSchemaImport/dto/EnumSchemaNames.ts

```diff
@@ -2,6 +2,9 @@ import { registerEnumType } from "@nestjs/graphql";
 
 export enum EnumSchemaNames {
```

Comment:
Do we need to re-define it outside of the models again?

Reply:
I mean can we only do the registerEnumType here and bring the type from models.ts?

Reply:
It wil not be in the models without having it here with the registerEnumType